src/data/location.json

```diff
@@ -9,40 +9,29 @@
     "forwardMarker": {
       "to": 2,
       "position": { "x": 2026, "y": -771, "z": 4503 },
+	  "lookAt": { "x": -12, "y": 53, "z": 4997 },
       "scale": 300
     },
     "backMarker": {
       "to": 20,
       "position": { "x": 4987.46, "y": -160.92, "z": -118.55 },
-      "scale": 300
+      "scale": 0.1
     },
-    "navMarkers": [
+    "nonavMarkers": [
```

Comment:
We should remove these now, I don't think we'll need them later and it will just confuse us